### PR TITLE
[React] added basic testing guide.

### PR DESCRIPTION
**Pull request by SKOLZ**
Added a basic guide on how to test using jest and enzyme

### CODE REVIEW

**Comment by mvbattan** [comment]
@SKOLZ Please follow the PR naming convention described in https://github.com/Wolox/tech-guides/pull/90.
Thanks !

**Comment by mvbattan** [comment]
It would be great to include reducer / actions testing as described in https://redux.js.org/docs/recipes/WritingTests.html
What do you think ?

**Comment by SKOLZ** [comment]
@mvbattan I think the redux tests are a great idea, the only "problem" is that this will be used for trainings and when new woloxers read this they won't have learnt redux yet. we should add that information though, I'm not sure if we should add that link directly in the training, a link here or a link directly in the react readme. what do you think?

File: frontend/react/docs/jest-enzyme-testing.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  );
+
+  describe('onPreviousPressed', () => {
+    describe('when currentIndex is greater than 0', () => {
+      it('decrements the currentIndex', () => {
+        renderedComponent.setState({
+          currentIndex: 1,
+        });
+        renderedComponent.instance().onPrevPressed();
+        expect(renderedComponent.state('currentIndex')).toEqual(0);
```
Comment (pabloferro) [inline frontend/react/docs/jest-enzyme-testing.md]: I'd move all "set-up" code to the `describe`, and leave only the expect in the `it`.

**Comment by mvbattan** [comment]
> the only "problem" is that this will be used for trainings
@SKOLZ Mmm, in that case we may include that link for future reference. Although new woloxer won't have learnt redux yet, IMHO including this reference at least as a link could be useful in the future.
In that case, while training a new woloxer I'd mention that there are redux recipes for testing, and make these as optional training material for reading.

**Comment by SKOLZ** [comment]
@mvbattan I'll add it at the end with the other links for any new woloxer who wants to learn about it or any other person that checks this documient

**Comment by gussiciliano** [comment]
@mishuagopian

**Pull request merged**